### ISSUE TITLE
sitemap.xml as a plain text - bug in declaration in urlset

### ISSUE THREAD

**Issue by poizon** [gohugoio/hugo#10515]
### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
hugo v0.104.3-58b824581360148f2d91f5cc83f69bd22c1aa331+extended linux/amd64 BuildDate=2022-10-04T14:25:23Z VendorInfo=gohugoio
</pre>

### Does this issue reproduce with the latest release?
I found bug in default sitemap template (for multilingial sites)
After generate sitemap (i.e https://example.org/en/sitemap.xml) and open it in browser - you see xml as a plain text, because urlset declaration is not valid 

### Solution
You need add to sitemap.xml template in urlset tag this declarations:
`<urlset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd http://www.w3.org/TR/xhtml11/xhtml11_schema.html http://www.w3.org/2002/08/xhtml/xhtml1-strict.xsd" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/TR/xhtml11/xhtml11_schema.html">`

**Comment by jmooring** [comment]
The problem is a 301 redirect with:

```
http://www.w3.org/1999/xhtml
```

```
wget http://www.w3.org/1999/xhtml
```

> --2022-12-09 15:20:18--  http://www.w3.org/1999/xhtml
Resolving www.w3.org (www.w3.org)... 104.18.23.19, 104.18.22.19, 2606:4700::6812:1613, ...
Connecting to www.w3.org (www.w3.org)|104.18.23.19|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://www.w3.org/1999/xhtml [following]
--2022-12-09 15:20:18--  https://www.w3.org/1999/xhtml
Connecting to www.w3.org (www.w3.org)|104.18.23.19|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://www.w3.org/1999/xhtml/ [following]
--2022-12-09 15:20:18--  https://www.w3.org/1999/xhtml/
Reusing existing connection to www.w3.org:443.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/html]
Saving to: ‘xhtml’

**Comment by jmooring** [comment]
@idarek The PR I submitted (#10516) fixes this trivial redirect problem, and adheres to both the sitemap protocol and [Google's recommendations for multilingual sites](https://developers.google.com/search/docs/specialty/international/localized-versions).

**Comment by arifke254** [comment]
Works locally. Fails in production (Cloudflare Pages).

**Comment by idarek** [comment]
> Works locally. Fails in production (Cloudflare Pages).

Work on both on my end. I copied templates from Hugo GitHub repo and created in my layout all works fine now.

**Comment by arifke254** [comment]
I copied the templates afresh. Still fails in production. But seeking CF advice on this.

**Comment by McShelby** [comment]
I also commented on the [changeset](https://github.com/gohugoio/hugo/commit/3fd0b78498597ceb343b7fda2e9b652f3e957478):

A XML namespace is usually not meant for browsing. The expected value should be using a `http://` prefix as mentioned in the [docs](https://www.sitemaps.org/protocol.html).

Switching to `https://` may result in unexpected behaviour (eg. tools that consume those XML files may have trouble with this).

**Comment by jmooring** [comment]
@McShelby 

I agree. Given that the host + path is unique, I incorrectly assumed that the protocol was irrelevant. Looking at the [spec](https://www.w3.org/TR/2006/REC-xml-names11-20060816/#NSNameComparison), it is indeed a full string comparison.

John Mueller of Google, in [this thread](https://webmasters.stackexchange.com/questions/108855/is-incorrect-to-have-the-https-version-of-the-sitemaps-org-url-in-the-xmlns-site), states:

> Yes, it's essentially an identifier. Google accepts both, since people tend to use them interchangeably nowadays.

But just because Google does, it does not mean that everyone else does. I will revert https://github.com/gohugoio/hugo/commit/3fd0b78498597ceb343b7fda2e9b652f3e957478.

In the future, please open a new issue instead of commenting on a commit or a closed issue. Thanks.
